torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -326,6 +326,22 @@ def __init__(self, target, args, kwargs):
         if target == torch.ops.aten.detach.default:
             return args[0]
 
+        if target == torch.ops.aten.as_strided.default:
+            unpack = lambda tensor, size, stride, offset=0: (tensor, size, stride, offset)
+            tensor, size, stride, offset = unpack(*args)
+            ndims = len(size)
+
+            if offset != 0 or len(stride) != ndims:
+                return self.call_function_prop_meta(target, args, kwargs)
```

Comment:
Why do we do `return None` in some other cases?

Reply:
I make returning `None` mean skipping conversion in `rewrite_node` nested in `ReplaceMoreTtManually`.  We are in `ReplaceMoreTt.call_function` here, so we skip conversion this way.